frontend/packages/helm-plugin/src/catalog/utils/catalog-utils.tsx

```diff
@@ -88,6 +111,7 @@ export const normalizeHelmCharts = (
             name,
             chartRepositoryTitle,
             version,
+            providerType: _.capitalize(providerType),
```

Comment:
We should localize the well known values.

Reply:
Done.